prime-angular-frontend/src/app/modules/phsa-eforms/pages/available-access/available-access.component.ts

```diff
@@ -44,7 +47,10 @@ export class AvailableAccessComponent implements OnInit {
   public onSubmit(): void {
     if (this.formUtilsService.checkValidity(this.form)) {
       const payload = this.phsaEformsFormStateService.json;
-      this.busy = this.phsaEformsResource.createEnrollee(payload)
+      this.busy = of(noop())
```

Comment:
GIS/PHSA are quick enrolment workflows with very little processing that we maintain, but aren't "owned" by us like our enrolment, paper enrolment, site reg, HA site reg, and admin.  I wouldn't update them until it's requested.

Reply:
Will remove PHSA and GIS